install.sh

```diff
@@ -3388,7 +3387,11 @@ search_for_templates_dir() {
 search_pre_commit_sample_file() {
     START_DIR="$1"
 
-    for HIT in $(find "$START_DIR" 2>/dev/null | grep "templates/hooks/pre-commit.sample"); do
+    IFS="
+"
+    # shellcheck disable=SC2044
+    for HIT in $(find "$START_DIR" -path "*templates/hooks/pre-commit.sample*" 2>/dev/null); do
```

Comment:
Nit: this would match `pre-commit.sample.whatever` too. I don't think the trailing `*` hurts the `find` performance too much, but I'd drop it.